lib/logstash/inputs/syslog.rb

```diff
@@ -136,6 +138,7 @@ def udp_listener(output_queue)
 
     while !stop?
       payload, client = @udp.recvfrom(9000)
+      metric.increment(:messages_received)
```

Comment:
I am feeling like `messages_received` isn't the right word here. Maybe `packets` ?

Let's also include bytes (`payload.bytesize`) as `bytes` metric maybe? Thoughts